lambda/src/main/scala/pricemigrationengine/services/CohortStateMachineLive.scala

```diff
@@ -0,0 +1,39 @@
+package pricemigrationengine.services
+
+import java.time.LocalDate
+
+import com.amazonaws.regions.Regions.EU_WEST_1
+import com.amazonaws.services.stepfunctions.AWSStepFunctionsClientBuilder
+import com.amazonaws.services.stepfunctions.model.{StartExecutionRequest, StartExecutionResult}
+import pricemigrationengine.model._
+import upickle.default.write
+import zio.{IO, ZIO, ZLayer}
+
+object CohortStateMachineLive {
+
+  val impl: ZLayer[CohortStateMachineConfiguration with Logging, ConfigurationFailure, CohortStateMachine] = {
+
+    val stateMachine = AWSStepFunctionsClientBuilder.standard.withRegion(EU_WEST_1).build
+
+    ZLayer.fromServicesM[CohortStateMachineConfiguration.Service,
+                         Logging.Service,
+                         Any,
+                         ConfigurationFailure,
+                         CohortStateMachine.Service] { (configuration, logging) =>
+      configuration.config map { config =>
+        new CohortStateMachine.Service {
+          def startExecution(date: LocalDate)(spec: CohortSpec): IO[CohortStateMachineFailure, StartExecutionResult] =
+            ZIO
```

Comment:
I keep meaning to start using effectBlocking for all of these calls to the aws client codes. Its not a problem as there is not really any parallelism, I might go through and fix it at some point.

Reply:
That would be an improvement.  I'll add it in this case.